package.json

```diff
@@ -12,19 +12,16 @@
     "@azure-tools/typespec-azure-resource-manager": "0.31.0",
     "@azure-tools/typespec-client-generator-core": "0.31.0",
     "@azure-tools/typespec-providerhub": "0.31.0",
-    "@azure-tools/typespec-validation": "file:eng/tools/TypeSpecValidation",
     "@typespec/compiler": "0.45.2",
     "@typespec/http": "0.45.0",
     "@typespec/openapi": "0.45.0",
     "@typespec/rest": "0.45.0",
     "@typespec/versioning": "0.45.0",
     "@azure/avocado": "^0.8.4",
     "@types/prettier": "^2.7.2",
+    "azure-rest-api-specs-eng-tools": "file:eng/tools",
```

Comment:
If the package.json is copied to another location will this reference fail? I would assume this is a relative path reference that would likely fail still.

Reply:
If a dependency points to a path that doesn't exist, `npm` just ignores the dependency with no warning or error.  However, if a `postinstall` script fails (say by trying to use a path that doesn't exist), then `npm` does fail.

So this PR fixes the original problem where some tools copy the root `package.json` to spec folders, and also works without requiring users to do `npm run build`.

Reply:
OK that definitely seems like a whole in npm but I'm fine with depending on it for now.